common/src/main/java/org/apache/nemo/common/dag/Edge.java

```diff
@@ -64,6 +64,15 @@ public final Integer getNumericId() {
     return Integer.parseInt(id.replaceAll("[^\\d.]", ""));
   }
 
+  /**
+   * Method to restore String ID from the numeric ID.
+   * @param numericId the numeric id.
+   * @return the restored string ID.
+   */
+  public static final String restoreId(final Integer numericId) {
```

Comment:
This is an utility function, I think it would be good to place this method into utility class (`Utils` or `DagUtils`? )

Reply:
Good idea.